runtime/laos/src/configs/evm.rs

```diff
@@ -47,9 +47,13 @@ parameter_types! {
 	///     (max_extrinsic.ref_time() / max_extrinsic.proof_size()) / WEIGHT_PER_GAS
 	/// )
 	pub const GasLimitPovSizeRatio: u64 = 4;
+	/// The amount of gas per storage (in bytes): BLOCK_GAS_LIMIT / BLOCK_STORAGE_LIMIT
+	/// The current definition of BLOCK_STORAGE_LIMIT is 160 KB, resulting in a value of 91.
+	pub const GasLimitStorageGrowthRatio: u64 = 91;
```

Comment:
where this number comes from ?

Reply:
it is described in the comment BLOCK_GAS_LIMIT / BLOCK_STORAGE_LIMIT (ie: 15_000_000 / 160 * 1024)

Reply:
https://github.com/freeverseio/laos/blob/1188ad142714a0af2a624b09fdfdd3504a9e2f87/runtime/laos/src/configs/evm.rs#L297-L298